setup.py

```diff
@@ -274,19 +276,18 @@ def get_vllm_version() -> str:
 
 
 def read_readme() -> str:
-    """Read the README file if present."""
-    p = get_path("README.md")
-    if os.path.isfile(p):
-        return io.open(get_path("README.md"), "r", encoding="utf-8").read()
-    else:
-        return ""
+    """Read the README file."""
+    return io.open(get_path("README.md"), "r", encoding="utf-8").read()
 
 
 def get_requirements() -> List[str]:
     """Get Python package dependencies from requirements.txt."""
     if _is_hip():
         with open(get_path("requirements-rocm.txt")) as f:
             requirements = f.read().strip().split("\n")
+    elif _is_cuda():
```

Comment:
requirements-cuda.txt does not exist in the repo. Besides - can we leave requirements.txt as is, and add separate path for handling HPU? We're making HPU the new default here.

Reply:
Added requirements-hpu.txt and left the default requirements.txt.